packages/tools/examples/nonVolumeNavigationTools/index.ts

```diff
@@ -0,0 +1,275 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  getRenderingEngine,
+  CONSTANTS,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  setTitleAndDescription,
+  addButtonToToolbar,
+  addDropdownToToolbar,
+  camera as cameraHelpers,
+} from '../../../../utils/demo/helpers';
+
+import vtkActor from '@kitware/vtk.js/Rendering/Core/Actor';
+import vtkSphereSource from '@kitware/vtk.js/Filters/Sources/SphereSource';
+import vtkMapper from '@kitware/vtk.js/Rendering/Core/Mapper';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  PanTool,
+  RotateTool,
+  ZoomTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+} = cornerstoneTools;
+
+const { ViewportType } = Enums;
+const { MouseBindings } = csToolsEnums;
+const { ORIENTATION } = CONSTANTS;
+
+const renderingEngineId = 'myRenderingEngine';
+const viewportId1 = 'POLYDATA_SAGITTAL_ORTHO';
+const viewportId2 = 'POLYDATA_SAGITTAL_STERO';
+const viewportIds = [viewportId1, viewportId2];
+
+// ======== Set up page ======== //
+setTitleAndDescription(
```

Comment:
can we rename this file to Manipulation Tools to be consistent?

Reply:
Also the whole `nonVolumeActor` is kind of strange to me, do we mean any non-volume Actor? probably not? So I would suggest to rename this example and the one in the core to be `polyDataActor` ??